### DISCUSSION TITLE
How to see measure descriptions in the UI?

### DISCUSSION THREAD

**Discussion by EricDallAgnol** [atoti/atoti#782] [Q&A] [answered]
Hi there 👋 

I have a question regarding the [`description` attribute](https://docs.atoti.io/latest/lib/atoti/atoti.measure.html#atoti.Measure.description).
Right now if I'm not wrong, there is no way to display that in the UI, but from what I've seen in [another discussion](https://github.com/atoti/atoti/discussions/771) this is on the roadmap am I correct ? If so, do you have an idea when we could have such feature in the UI ?

_We have a lot of measures filtering on several dimensions and it's not easy for users from the measure name to guess that, hence it would be really nice to be able to provide a concise description directly in the UI for each measure, using this attribute !_

Thank you,
Eric

**Comment by tibdex** [comment, marked as answer]
Hi Eric,

This will be available in the next release (planned in May):

![Screenshot 2023-04-27 at 12 08 51](https://user-images.githubusercontent.com/6574550/234921881-deb13ad2-ec2d-4126-b9ed-788b3d843ef6.png)

**Reply by EricDallAgnol** [reply]
Hi Thibault,

Perfect thank you for your answer !
